base_app/src/app/section/section.component.html

```diff
@@ -54,9 +54,10 @@
             </prop-defined-table>
 
             <app-smart-table *ngIf="getTableType(dsItem) === 'smart'"
```

Comment:
I wonder if this should be called `app-smart-fault-table`, since it's specific to faults.